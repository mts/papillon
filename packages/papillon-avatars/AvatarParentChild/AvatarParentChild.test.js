import avatarParentChild from './AvatarParentChild.render'

describe('<Avatar />', () => {
  let wrapper

  describe('Instance', () => {
    test('must be an instance of AvatarParentChild', () => {
      wrapper = renderMount(avatarParentChild)
      expect(wrapper.find('AvatarParentChild')).toHaveLength(1)
    })
  })

  describe('Snaphot', () => {
    test('must match avatarParentChild', () => {
      expect(global.renderToJSON(avatarParentChild)).toMatchSnapshot()
    })
  })
})
