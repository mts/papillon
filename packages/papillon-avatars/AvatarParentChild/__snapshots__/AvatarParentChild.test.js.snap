// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Avatar /> Snaphot must match avatarParentChild 1`] = `
<div
  className="avatar-parent-child float-left"
>
  <img
    alt="parent"
    className="avatar"
    height={48}
    src="https://user-images.githubusercontent.com/334891/29999089-2837c968-9009-11e7-92c1-6a7540a594d5.png"
    width={48}
  />
  <img
    alt="child"
    className="avatar avatar-child"
    height={20}
    src="https://user-images.githubusercontent.com/334891/29999089-2837c968-9009-11e7-92c1-6a7540a594d5.png"
    width={20}
  />
</div>
`;
